app/models/manageiq/providers/automation_manager.rb

```diff
@@ -32,4 +34,6 @@ def total_configuration_profiles
   def total_configured_systems
     Rbac.filtered(configured_systems, :match_via_descendants => ConfiguredSystem).count
   end
+
+  delegate :url, to: :provider
```

Comment:
Prefer `delegate :url, :to => :provider` for consistency.  Also, most of our delegations are at the top of the file.